etc/migration/10.3-to-11.0/migration.cql3

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "Principals" ADD "emailChange" text;
+ALTER TABLE "Principals" ADD "emailVerified" boolean;
```

Comment:
I thought we discussed that the existence of an email in the `email` field was going to imply that it is verified. No additional column would be necessary.

Also, it's possible that we wouldn't need an `emailChange` property on the principal profile if we would be fine with only showing the "you have an updated email address that needs to be verified" in the user profile modal. If we can verify that w/ @nicolaasmatthijs, then we can store this in a separate table and not add more cruft to Principals.